### PR TITLE
Fix #18

### LINKED ISSUE #18: --forwrev should be non-zero by default
Setting `forwrev` to 0 results in an empty dataframe in [forward_backward_convergence](https://github.com/alchemistry/alchemlyb/blob/0916cba61d141e4da171c6891edb9f65aad51ac1/src/alchemlyb/convergence/convergence.py#L15) which breaks other stuff, making `flamel` unusable unless `--forwrev` != 0 is specified.

### PR DESCRIPTION
**Pull request by xiki-tempula**
## Description
Handle the case where forrev is 0. Fix #18
Merge after #21
## Status
- [ ] Ready to go

### CODE REVIEW

**Comment by xiki-tempula** [comment]
@hejamu Many thanks for the PR, I think defaulting the parameter to 10 might still cause the problem if the user chooses to have this value as 0 so I fixed the issue here. Do you mind if I add you as co-author for this fix?

File: CHANGES [hunk 1]
```diff
@@ -14,6 +14,14 @@ The rules for this file:
 
 ------------------------------------------------------------------------------
 
+2022-xx-xx  xiki-tempula, Henrik Jaeger
```
Comment (xiki-tempula) [inline CHANGES:17]: @hejamu I mean here

**Comment by hejamu** [comment]
Agreed. This feels like the most sensible behavior

File: CHANGES (hunk 1 above)
Comment (hejamu) [reply CHANGES:17]: I have no issue with it :)

Comment (orbeckst) [reply CHANGES:17]: I'd suggest to use the GitHub handle "hejamu" instead of clear names, simply because that's what we've been doing everywhere.

Comment (orbeckst) [reply CHANGES:17]:
```suggestion
2022-xx-xx  xiki-tempula, hejamu
```

**Review by orbeckst — APPROVED** [review]
lgtm, I'll just change the CHANGES entry before merging — thanks!

**Pull request merged**